Mathlib/LinearAlgebra/UnitaryGroup.lean

```diff
@@ -185,6 +188,22 @@ theorem mem_specialUnitaryGroup_iff :
     A ∈ specialUnitaryGroup n α ↔ A ∈ unitaryGroup n α ∧ A.det = 1 :=
   Iff.rfl
 
+instance : StarMul (specialUnitaryGroup n α) where
+  star A := ⟨star A, by simpa using A.prop.1, by have := A.prop.2; simp_all [star_eq_conjTranspose]⟩
+  star_mul A B := Subtype.ext <| star_mul A.1 B.1
+  star_involutive A := Subtype.ext <| star_involutive A.1
+
+@[simp, norm_cast]
+theorem specialUnitaryGroup.coe_star (A : specialUnitaryGroup n α) : (star A).1 = star A.1 := rfl
+
+instance : Inv (specialUnitaryGroup n α) where inv := star
+
+theorem star_eq_inv (A : specialUnitaryGroup n α) : star A = A⁻¹ :=
+  rfl
+
+instance : Group (specialUnitaryGroup n α) where
+  inv_mul_cancel A := Subtype.ext A.prop.1.1
```

Comment:
Can you add the map to `unitaryGroup` as a homomorphism?

Reply:
I decided against that previously because it's just `Submonoid.inclusion`. I added the inequality instead.